BUILD.gn

```diff
@@ -1,958 +1,1024 @@
-    }
-  }
-}
-
-group("chromium_browsertests") {
-  testonly = true
-
-  deps = [
-    "//content/test:content_browsertests",
-  ]
-}
-
-template("dist_zip") {
-  _runtime_deps_target = "${target_name}__deps"
-  _runtime_deps_file =
-      "$root_out_dir/gen.runtime/" + get_label_info(target_name, "dir") + "/" +
-      get_label_info(target_name, "name") + ".runtime_deps"
-
-  group(_runtime_deps_target) {
-    forward_variables_from(invoker,
-                           [
-                             "deps",
-                             "data_deps",
-                             "data",
-                           ])
-    write_runtime_deps = _runtime_deps_file
-  }
-
-  action(target_name) {
-    script = "//electron/build/zip.py"
-    deps = [
-      ":$_runtime_deps_target",
-    ]
-    forward_variables_from(invoker, [ "outputs" ])
-    args = rebase_path(outputs + [ _runtime_deps_file ], root_build_dir) + [
-             target_cpu,
-             target_os,
-           ]
-  }
-}
-
-copy("electron_license") {
-  sources = [
-    "LICENSE",
-  ]
-  outputs = [
-    "$root_build_dir/{{source_file_part}}",
-  ]
-}
-copy("chromium_licenses") {
-  deps = [
-    "//components/resources:about_credits",
-  ]
-  sources = [
-    "$root_gen_dir/components/resources/about_credits.html",
-  ]
-  outputs = [
-    "$root_build_dir/LICENSES.chromium.html",
-  ]
-}
-
-group("licenses") {
-  data_deps = [
-    ":electron_license",
-    ":chromium_licenses",
-  ]
-}
-
-copy("electron_version") {
-  sources = [
-    "VERSION",
-  ]
-  outputs = [
-    "$root_build_dir/version",
-  ]
-}
-
-dist_zip("electron_dist_zip") {
-  data_deps = [
-    ":electron_app",
-    ":licenses",
-    ":electron_version",
-  ]
-  outputs = [
-    "$root_build_dir/dist.zip",
-  ]
-}
-
-dist_zip("electron_ffmpeg_zip") {
-  data_deps = [
-    "//third_party/ffmpeg",
-  ]
-  outputs = [
-    "$root_build_dir/ffmpeg.zip",
-  ]
-}
-
-dist_zip("electron_chromedriver_zip") {
-  data_deps = [
-    "//chrome/test/chromedriver",
-    ":licenses",
-  ]
-  outputs = [
-    "$root_build_dir/chromedriver.zip",
-  ]
-}
-
-dist_zip("electron_mksnapshot_zip") {
-  data_deps = [
-    "//v8:mksnapshot($v8_snapshot_toolchain)",
-    ":licenses",
-  ]
-  outputs = [
-    "$root_build_dir/mksnapshot.zip",
-  ]
-}
-
-group("electron") {
-  deps = [
-    ":electron_app",
-  ]
-}
-
-group("electron_content_manifest_overlays") {
-  deps = [
-    ":electron_content_browser_manifest_overlay",
-    ":electron_content_packaged_services_manifest_overlay",
-  ]
-}
-
-service_manifest("electron_content_packaged_services_manifest_overlay") {
-  source = "//electron/manifests/electron_content_packaged_services_manifest_overlay.json"
-  packaged_services = [ "//services/proxy_resolver:proxy_resolver_manifest" ]
-}
-
-service_manifest("electron_content_browser_manifest_overlay") {
-  source = "//electron/manifests/electron_content_browser_manifest_overlay.json"
-}
+import("build/asar.gni")

```

Comment:
@brenca I think you committed the CLRF line endings for this file instead of just LF